models/reviews.js

```diff
@@ -0,0 +1,31 @@
+const db = require("../db/connection");
+const { getCommentCount } = require("../utils");
+
+exports.getReviews = () => {
+    let commentCount;
+    return getCommentCount()
```

Comment:
I like that you've extracted this out into a utils function, but beyond that 1) this could be a single SQL Query as opposed to doing 2 separate DB queries and 2) we are using a lot of JavaScript here for something that could essentially be done by utilizing an SQL query

Take a look at the SQL Relationship notes (last example): https://notes.northcoders.com/courses/js-back-end/sql-relationships